src/store.ts

```diff
@@ -14,12 +14,12 @@ const rootReducer = combineReducers({
 
 export const setupStore = (preloadedState?: PreloadedState<RootState>) => {
   return configureStore({
-    reducer: rootReducer,
-    preloadedState,
     middleware: getDefaultMiddleware =>
       getDefaultMiddleware({ immutableCheck: true }).concat(
         mentorsApi.middleware,
       ),
+    preloadedState,
+    reducer: rootReducer,
```

Comment:
Is it necesarry here to have the properties in alphabetical order?
I would argue that its better to have most relevant in the top, but this is just a preference

Reply:
Also take this comment just as an opinion

Reply:
I agree with you that for some of these objects the best order is not alphabetization. But I still think that enforcing some consistant ordering improves readability and reduces the time to find a specific key.

We could eslint-disable `sort-keys` for some files, but it would be hard to enforce some other order.

What do you think?

Reply:
I'm a bit torn on this too for API requests etc. But for stuff like the list of colors, different component variants or our own objects this rule is super useful.

Reply:
I dont know, is it not possible to only alphabetize the styled-components?

Reply:
Yes of course but I also like the alphabetization for variable lists like this 👇
![Screenshot 2023-01-27 at 11 17 08](https://user-images.githubusercontent.com/35532568/215050821-391b3fc8-c98b-4df3-b840-a64e904a49c0.png)

Reply:
The styled-components rule uses its own plugin.

Reply:
My opinion is that we dont need to enforce this, because it does not make sense in a lot of other cases. =P 
It might make the code even more difficult to read. In the end a variable list not being alphabetized is not that big of an issue I think